### DISCUSSION TITLE
Bei erhöhter Anzahl von Bildern -> Fehler

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1918] [Q&A] [answered]
Hat man viele Bilder zu laden auf einer Seite, kommt es manchmal vor, dass Meldungen angezeigt werden.
![image](https://user-images.githubusercontent.com/84818934/192629901-111fb278-1d77-4a4a-9b6c-ffa99d58d8ab.png)
![image](https://user-images.githubusercontent.com/84818934/192629989-0061411a-9c86-464b-a744-988ea2fc960e.png)

Nach einige Male F5 sind die Bilder aber da. Kann man da mit timeout-Werten arbeiten?

**Comment by Zefau** [comment]
Wie hast du die Bilder eingebettet? Glaube nicht, dass ich da was machen kann.

Der Fehler 503 (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503) wird erzeugt, weil du kurzfristig zu viele Anfragen an den Ziel-Server schickst.

**Comment by Zefau** [comment, marked as answer]
Versuch mal mit `loading="lazy"`, siehe https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img?retiredLocale=de#attr-loading

**Reply by mcuiobroker** [reply]
Läuft besser. Danke.
